portality/api/v2/crud/applications.py

```diff
@@ -58,6 +61,20 @@ def create(cls, data, account, dry_run=False):
             ia = IncomingApplication(data)
         except seamless.SeamlessException as e:
             raise Api400Error(str(e))
+        except dataobj.ScriptTagFoundException as e:
+            jdata = json.dumps(data, indent=4)
```

Comment:
at this stage do we have enough useful information in the article data itself or do we need to add more? (account or API key perhaps)

Reply:
I added the account information to the data in the email.